build/ci/phase-template.yml

```
@@ -14,12 +15,15 @@ phases:
       timeoutInMinutes: 45
       parallel: 99
       matrix:
-        Build_Debug:
-          _configuration: Debug
-          _config_short: D
-        Build_Release:
-          _configuration: Release
-          _config_short: R
+        ${{ if eq(parameters.customMatrixes, '') }}:
+          Build_Debug:
+            _configuration: Debug
+            _config_short: D
+          Build_Release:
+            _configuration: Release
+            _config_short: R
+        ${{ if ne(parameters.customMatrixes, '') }}:
+          ${{ insert }}: ${{ parameters.customMatrixes}}
```

Could you add a space at the end in between customMatrixes and }} ? 
```suggestion
          ${{ insert }}: ${{ parameters.customMatrixes }}
```